src/mvba/abba/message.rs

```diff
@@ -2,18 +2,27 @@ use blsttc::{Signature, SignatureShare};
 use serde::{Deserialize, Serialize};
 
 #[derive(Debug, Clone, Copy, Serialize, Deserialize, Eq, PartialEq)]
-pub enum PreVoteValue {
+pub enum Value {
```

Comment:
`PreVoteValue` make more seance than `Value`. `MainVoteValue` and `PreVoteValue` are two different things even the actual value look same.

Reply:
Whether it's wrapped in MainVotes or PreVotes, we're still talking about the same value `b \in {0,1}`.

Reply:
Sounds Good.